stmt/stmts/create_test.go

```diff
@@ -32,18 +33,20 @@ var _ = Suite(&testStmtSuite{})
 type testStmtSuite struct {
 	dbName string
 
-	testDB         *sql.DB
-	createDBSql    string
-	dropDBSql      string
-	useDBSql       string
-	createTableSql string
-	insertSql      string
-	selectSql      string
+	testDB                  *sql.DB
+	createDBSql             string
+	dropDBSql               string
+	useDBSql                string
+	createTableSql          string
```

Comment:
Yes, SQL is structured query language, it doesn't means text.
But we do mean text here.